apps/balancer-tools/src/app/apr/round/(components)/PoolListTable.tsx

```diff
@@ -145,20 +152,27 @@ function TableRow({
   network,
   tableData,
   setTableData,
+  throttleHandler,
 }: {
   poolId: string;
   roundId: string;
   network: number;
   tableData: PoolTableData[];
   setTableData: Dispatch<SetStateAction<PoolTableData[]>>;
+  throttleHandler: typeof throttle;
 }) {
   const pool = new Pool(poolId);
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
     const fetchData = async () => {
+      const throttledFn = throttleHandler(
+        async (poolId: string, roundId: string) => {
+          return await fetcher(`/apr/api?poolid=${poolId}&roundid=${roundId}`);
```

Comment:
this call doesn't need to be throttled

Reply:
I think it should be because they're going to be made alongside fetches from the `TopPoolsChart` component

Reply:
but if they're being handed to our own internal API that is going to cache these values that doesn't make a lot of sense -- also we're only showing 10 per page, so not a huge amount of requests to be doing at the same time.